components/specification/released-schema/2016-DEV0/ome.xsd

```diff
@@ -3778,6 +3777,9 @@ If Offset and ReadLength are specified in conjuction with Compression, then they
                         </xsd:documentation>
                 </xsd:annotation>
                 <xsd:complexType>
+                                        <xsd:complexContent>
```

Comment:
Could this section be reindented (maybe something for a followup PR for the entire file, if it's inconsistent).